source/tests/pt_expt/model/test_ener_hessian_model.py

```
@@ -0,0 +1,220 @@
+
+        # compute finite-difference hessian as reference
+        fnt_hess = []
+        for ii in range(nf):
+
+            def np_infer(
+                xx,
+            ):
+                xx_t = to_torch_array(xx).unsqueeze(0).requires_grad_(True)
+                ret = self.model_valu.forward(
+                    xx_t,
+                    atype[ii].unsqueeze(0),
```

Ruff (0.15.2)

[warning] Function definition does not bind loop variable `ii`

(B023)